### PR TITLE
feat(airdrop): Add script to remove principals from airdrop 

### PR DESCRIPTION
**Pull request by EnzoPlayer0ne**

### CODE REVIEW

File: scripts/airdrop.remove-principals.sh [hunk 1]
```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+# Get list of principals to remove from the airdrop
+
+set -eExuo pipefail
```
Comment (peterpeterparker) [inline scripts/airdrop.remove-principals.sh:5]: remove

File: scripts/airdrop.remove-principals.sh [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+            IFS=',' read -ra principals <<< "${1#*=}"
+            ;;
+        *)
+            print_usage_and_exit "Invalid argument - $1"
+            ;;
+    esac
+    shift
+done
+
+case $ENV in
```
Comment (peterpeterparker) [inline scripts/airdrop.remove-principals.sh:28]: we have this now in multiple scripts, not a prio but if one of these days you have time to refactor it, would be neat

Comment (anedos-dfinity) [reply scripts/airdrop.remove-principals.sh:28]: yeah, most the scripts by now have a lot of commonality

**Review by anedos-dfinity — APPROVED** [review]
Thanks @EnzoPlayer0ne !

File: scripts/airdrop.remove-principals.sh (hunk 1 above)
Comment (EnzoPlayer0ne) [reply scripts/airdrop.remove-principals.sh:5]: For script that change the state and are not called during deployment of the canister I like to see the command that is effectively being run!

File: scripts/airdrop.remove-principals.sh (hunk 2 above)
Comment (EnzoPlayer0ne) [reply scripts/airdrop.remove-principals.sh:28]: I agree - I can take a look tomorrow - also we should read the wallet from file ideally!

File: scripts/airdrop.remove-principals.sh (hunk 1 above)
Comment (peterpeterparker) [reply scripts/airdrop.remove-principals.sh:5]: Oki doki

Comment (EnzoPlayer0ne) [reply scripts/airdrop.remove-principals.sh:5]: I kept only `set -x` to see the command and removed the rest!

**Pull request merged**